dependency_management/compare_fedora_versions

```diff
@@ -97,6 +130,17 @@ def main():
             )
             result["low"].append(crate)
 
+    ignore_missing = args.ignore_missing
+    result["missing"] = [
+        missing for missing in result["missing"] if missing not in ignore_missing
+    ]
+
+    ignore_high = args.ignore_high
+    result["high"] = [high for high in result["high"] if high not in ignore_high]
+
+    ignore_low = args.ignore_low
```

Comment:
Might as well get rid of these ```<x> = <y>``` assignments and just use ```<y>``` instead of ```<x>``` in the list comprehension.

Reply:
Resolved.